doc/input-output-reference/src/overview/group-internal-gains-people-lights-other.tex

```diff
@@ -560,6 +588,22 @@ \subsubsection{Outputs}\label{outputs-017}
 
 This field reports the ideal indoor operative temperature, or comfort temperature, as determined by the CEN-15251 adaptive comfort model. Category I, II, and II limits for indoor operative temperature are defined as no greater than 2, 3, and 4 degrees C from this value respectively. A value of -1 means not applicable (when running average outdoor temp is outside the range of 10.0 to 30.0C).
 
+\paragraph{Zone Thermal Comfort ASHRAE 55 Elevated Air Speed Cooling Effect}\label{zone-thermal-comfort-ashrae55-elevated-air-speed-cooling-effect}
+
+This field is the calculated Cooling Effect of the elevated air speed in degree celsius. It is the value that, when subtracted equally from both the average air temperature and the mean radiant temperature, yields the same SET under still air as in the first SET calculation under elevated air speed.
+
+\paragraph{Zone Thermal Comfort ASHRAE 55 Elevated Air Speed Cooling Effect Adjusted PMV}\label{zone-thermal-comfort-ashrae55-elevated-air-speed-cooling-effect-adjusted-pmv}
+
+This field is the \"predicted mean vote\" (PMV) calculated using the Fanger PMV model, adjusted by the ASHRAE 55 Elevated Air Speed Cooling Effect. The Cooling Effect adjusted PMV for an environment with elevated average air speed is calculated using the adjusted average air temperature, the adjusted radiant temperature, and still air (0.1 m/s). 
```

Comment:
predicted is rendering as p̈redicted . Use the smart-quote characters (see existing entries further up): ``predicted mean vote'' (2 characters before and after). Same problem in the next two descriptions.